dockerlib/Dockerfile

```diff
@@ -1,9 +1,10 @@
-FROM golang:1.9.2-stretch as builder
+FROM golang:1.15-alpine as builder
 COPY . /go/src/github.com/influxdata/influxdata-docker/dockerlib
-RUN set -xe && \
-    go get -d github.com/influxdata/influxdata-docker/dockerlib && \
+RUN set -xe; \
+    apk add --no-cache git; \
+    go get -d github.com/influxdata/influxdata-docker/dockerlib; \
     go install github.com/influxdata/influxdata-docker/dockerlib
 
-FROM buildpack-deps:stretch-scm
```

Comment:
This image isn't one pushed to official images. It's just for compiling this simple utility program. With go modules, this Dockerfile should probably just be removed.

Reply:
I just wanted to modernize it.